### PR TITLE
[13092289] Adjusted the login screen text.

### PR DESCRIPTION
**Pull request by jbest84**
@CHenrie Please review

### CODE REVIEW

**Comment by slxbuilder** [comment]
Merged build triggered.

**Comment by slxbuilder** [comment]
Merged build started.

**Comment by slxbuilder** [comment]
Merged build finished.

File: src/Views/Login.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ define('Mobile/SalesLogix/Views/Login', [
         logOnText: 'Log On',
         passText: 'password',
         rememberText: 'remember',
-        titleText: 'SalesLogix',
+        titleText: 'Saleslogix',
```
Comment (CHenrie) [inline src/Views/Login.js:27]: @jbest84  I thought the At SaleLogix is with a capital "L"

Comment (jbest84) [inline src/Views/Login.js:27]: @CHenrie No, the defect says to make it lowercase.

Comment (CHenrie) [inline src/Views/Login.js:27]: @jbest84 Oh So they are changing it. Because every where else it is and upper case.

**Comment by slxbuilder** [comment]
Merged build triggered.

**Comment by slxbuilder** [comment]
Merged build started.

**Comment by slxbuilder** [comment]
Merged build finished.

**Pull request merged**